.github/workflows/auto-approve.yml

```diff
@@ -17,6 +17,7 @@ jobs:
         startsWith(github.event.issue.title, 'publish: getsentry/sentry-forked-djangorestframework-stubs@') ||
         startsWith(github.event.issue.title, 'publish: getsentry/sentry-kafka-schemas@') ||
         startsWith(github.event.issue.title, 'publish: getsentry/sentry-redis-tools@') ||
+        startsWith(github.event.issue.title, 'publish: getsentry/statsdproxy@') ||
         startsWith(github.event.issue.title, 'publish: getsentry/snuba-sdk@') ||
```

Comment:
can you alphabetize these?

Reply:
https://github.com/getsentry/publish/pull/3232